spec/models/rating_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Rating, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :rating }
+    it { is_expected.to have_db_column :counter }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :rating }
+    it { is_expected.to validate_presence_of :counter }
+  end
+
+  describe 'Associations' do
+    it { is_expected.to belong_to :restaurant }
```

Comment:
Any dependancy on the parent? 
http://guides.rubyonrails.org/association_basics.html#belongs-to-association-reference